src/services/category.js

```
@@ -16,8 +16,13 @@
       Category.countDocuments(query)
     ]);
 
+    const categoriesWithOffers = categories.map(category => ({
+      ...category,
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)